test_utilities.py

```diff
@@ -0,0 +1,57 @@
+import utilities
+
+
+class TestClass:
+    test_input = "The quick brown fox jumps over the lazy dog."
+    clf = utilities.classify_model()
+
+    def test_setup_nltk(self):
+        result = utilities.setup_nltk()
+        assert result
+
+    def test_parse_sentence(self):
+        triples, root = utilities.parse_sentence(self.test_input)
+        triples = list(triples)
+        assert (("jumps", "VBZ"), "nsubj", ("fox", "NN")) in triples
+        assert (("jumps", "VBZ"), "nmod", ("dog", "NN")) in triples
+        assert root == "jumps"
+
+    def test_classify_model(self):
+        from features import features_dict
+        import hashlib
+        import numpy as np
+
+        keys = [
+            "id",
+            "wordCount",
+            "stemmedCount",
+            "stemmedEndNN",
+            "CD",
+            "NN",
+            "NNP",
+            "NNPS",
+            "NNS",
+            "PRP",
+            "VBG",
+            "VBZ",
+            "startTuple0",
+            "endTuple0",
+            "endTuple1",
+            "endTuple2",
+            "verbBeforeNoun",
+            "qMark",
+            "qVerbCombo",
+            "qTripleScore",
+            "sTripleScore",
+            "class",
+        ]
+        id = hashlib.md5(str(self.test_input).encode("utf-8")).hexdigest()[:16]
+        f = features_dict(id, self.test_input)
+        features = [f[k] for k in keys][1:-1]
+        features = np.array(features).reshape(1, -1)
```

Comment:
Creating features for the `test_input` after referring to logic from `featuresDump.py` and passing it to the classifier created to predict a class.

We already know the class of `test_input` is _Statement_ (`S`)